plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlScannerTest.java

```diff
@@ -64,4 +109,30 @@ public void testStartElement_correctVersion() throws SAXException {
     assertEquals(0, scanner.getBlacklist().size());
   }
   
+  @Test
+  public void testFindClass() {
+    assertFalse(WebXmlScanner.classExists(javaProject, "DoesNotExist"));
+    assertFalse(WebXmlScanner.classExists(null, null));
+    assertFalse(WebXmlScanner.classExists(null, ""));
+    assertTrue(WebXmlScanner.classExists(javaProject, "ServletClass"));
```

Comment:
As I said above, the following assertion will pass. (Not that you should change it.)

```java
assertTrue(WebXmlScanner.classExists(javaProject, "src.main.java.ServletClass"));
```